src/core/sys/freebsd/sys/event.d

```diff
@@ -28,10 +30,11 @@ enum
     EVFILT_PROC     =  -5, /* attached to struct proc */
     EVFILT_SIGNAL   =  -6, /* attached to struct proc */
     EVFILT_TIMER    =  -7, /* timers */
-    // EVFILT_NETDEV   =  -8, /* no longer supported */
+    EVFILT_PROCDESC =  -8, /* attached to process descriptors */
```

Comment:
Ah, I'm sorry, I've missed that it was already commented. Thank you for pointing that out!